### DISCUSSION TITLE
Train Station Boundary conditions

### DISCUSSION THREAD

**Discussion by FPSSPF** [firemodels/fds#13437] [Q&A]
Hello, 

I am preparing my master thesis and I have two doubts.

1 I would like to model a train station connected by means of tunnels to other stations. I would like to include the pressure and velocity wave produced by an incoming train. Is there any guidance or paper illustrating this?

2 Is there a well known guidance/paper/experiment regarding the Heat Release Rate of a burning train? 

Thanks in advance

**Comment by IFAB-AS** [comment]
Hello,

1) We did some measurements a few weeks ago at the tunnel sections at the end of a station and the following values are realistic assumptions: Without train movement: 0.2 - 0.5 m/s, with train movements: mostly peaks in range 1-4 m/s, but there can be peaks up to 8 m/s (may depend on whether there is a train at the other track or not or the speed of the incoming vehicle). We did measure in September, so there might be differences for other seasons. Also: the tunnel system itself can have slopes and different heights above zero at the entrances and various cross-sections, so this will also influence the flow.
The time it took to get back to the steady state was approx. 30-70 s.

2) Trains can have huge differences in HRR. E.g. the TRStrab curve ([TRIS database,](https://technical-regulation-information-system.ec.europa.eu/de/notification/20906,) final text only in German but the draft is also available in English) reaches 55.5 MW for a 40 m vehicle, other vehicles can reach even more. Some modern vehicles are very hard to ignite and might reach less than 2 MW. It depends on the materials that are used within the vehicle, the material at the ceiling in the vehicle, the ventilation conditions (fire in a tunnel, fire in the station, breaking of windows etc.) and also the ignition position and size of the fire that ignites the vehicle.
